dev_tools/modules.py

```diff
@@ -183,6 +183,7 @@ def _parse_module(folder: Path) -> Dict[str, Any]:
 
     orig_setup = setuptools.setup
     cwd = os.getcwd()
+    sys.path.append(cwd)
```

Comment:
Maybe `sys.path.insert(0, cwd)` ?

In case PYTHONPATH is set for another Cirq working tree, putting cwd first should guarantee the version value is from the current repository.

Reply:
Excellent point. I'll change it.